components/tidb_query/src/rpn_expr/impl_compare_in.rs

```diff
@@ -235,10 +235,10 @@ mod tests {
     use crate::codec::batch::{LazyBatchColumn, LazyBatchColumnVec};
     use crate::expr::EvalContext;
     use crate::rpn_expr::types::RpnFnMeta;
-    use crate::rpn_expr::RpnExpressionBuilder;
+    use crate::rpn_expr::{RpnExpressionBuilder, RpnExpressionNode};
 
     #[test]
-    fn test_in() {
+    fn test_in_constant() {
```

Comment:
Prefer to add a test that include both constants and non-constants.

Reply:
`test_in_complex` include both constants and non-constants, I think that a `test_in_constant` is more obvious to show the result.

Reply:
Then how about also check remaining constants in test_in_complex?

Reply:
done